src/amp-story-player/amp-story-player-impl.js

```diff
@@ -648,7 +648,20 @@ export class AmpStoryPlayer {
    * @private
    */
   next_() {
-    if (this.currentIdx_ + 1 >= this.stories_.length) {
+    if (
+      this.element_.getAttribute('enable-circular-wrapping') != 'true' &&
+      this.isIndexOutofBounds_(1)
+    ) {
+      return;
+    }
+
+    if (
+      this.element_.getAttribute('enable-circular-wrapping') === 'true' &&
```

Comment:
Since this line is used so many times, storing it as a variable might be good. Then you wouldn't have to check it against a string, you could just use `this.enableCircularWrapping` as a boolean.